packages/manager/src/components/TransferDisplay/TransferDisplay.tsx

```diff
@@ -68,145 +41,47 @@ export const TransferDisplay = React.memo(({ spacingTop }: Props) => {
   }
 
   return (
-    <>
-      <Typography
-        className={classes.root}
-        style={{ marginTop: spacingTop ?? 8 }}
-      >
+    <Box marginTop={spacingTop}>
+      <StyledTransferDisplayContainer>
         {isLoading ? (
-          'Loading transfer data...'
+          <>
+            <Typography>Loading transfer data...</Typography>
+            <CircleProgress mini />
+          </>
         ) : (
           <>
-            You have used {poolUsagePct.toFixed(poolUsagePct < 1 ? 2 : 0)}% of
-            your
-            {`  `}
-            <button
-              className={classes.openModalButton}
-              onClick={() => setModalOpen(true)}
-            >
-              Monthly Network Transfer Pool
-            </button>
-            .
+            <Typography>
+              <StyledLinkButton
+                aria-label="Show the Monthly Network Transfer Pool"
+                data-testid="open-transfer-display-modal-button"
+                onClick={() => setModalOpen(true)}
+              >
+                Monthly Network Transfer Pool
+              </StyledLinkButton>
+              &nbsp;usage:
+            </Typography>
+            <Typography data-testid="transfer-pool-pct-display">
+              {formatPoolUsagePct(generalPoolUsagePct)} Global Transfer Pool
+            </Typography>
+            {generalPoolUsage?.quota !== 0 && // Don't display region transfer pools if the general pool is empty (e.g. account has no resources).
```

Comment:
```suggestion
            {
```
We shouldn't need this conditional here. This results in the MNTP dialog not showing transfer data if a user has regional transfer in, for example, only Jakarta and no transfer in the global transfer pool, which is a plausible situation. The account may have resources, they could just be in datacenter-specific pools.

Reply:
you're right! Should have revisited after actually understanding the feature 🙄

Reply:
done!